packages/plugin-compat/extra/typescript/gen-typescript-patch.js

```diff
@@ -267,21 +288,24 @@ async function run({from, to, onto, range}) {
 }
 
 async function validate(version, patchFile) {
```

Comment:
WSL doesn't clean the `/tmp` folder so I had about 10GB of TypeScript there, the changes here should avoid that and improve the performance of repeated runs